CMakePresets.json

```diff
@@ -37,48 +35,33 @@
       }
     },
     {
-      "name": "x86-debug",
-      "displayName": "x86 Debug",
-      "description": "Target Windows Debug (32-bit).",
-      "inherits": "windows-base",
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug"
-      }
-    },
-    {
-      "name": "x86-release",
-      "displayName": "x86 Release",
-      "description": "Target Windows Release (32-bit).",
-      "inherits": "x86-debug",
+      "name": "x64-debinfo",
+      "displayName": "x64 Release with Debug info",
+      "description": "Target Windows Release with Debug info (64-bit).",
+      "inherits": "x64-debug",
```

Comment:
Shouldn't inherit from debug. Instead it should inherit from release whilst also keeping the debug symbols (it's an option i think)

Reply:
Release inherits from Debug anyway, just changing the release type but if it makes more sense to you this way, sure.